Guide.docc/IncrementalAdoption.md

```diff
@@ -149,6 +149,60 @@ class UIStyler {
 Combining static and dynamic isolation can be a powerful tool to keep the
 scope of changes gradual.
 
+## Missing Annotations
+
+Dynamic isolation gives you tools to express isolation at runtime.
+But, you may also find you need to describe other concurrency properties
+that are missing from unmigrated modules.
+
+### Unmarked Sendable Closures
+
+A callback that is missing a `Sendable` annotation can be a problem when
+combined with actor-isolated types.
+
+```swift
+@MainActor
+class PersonalTransportation {
+    func configure() {
+        // This function is missing a Sendable annotation
+        JPKJetPack.jetPackConfiguration {
+            // MainActor isolation will be inferred here, incorrectly
```

Comment:
```suggestion
            // MainActor isolation will be inferred here
```

It's hard to "know why incorrectly" here -- so maybe skip that.

Maybe when we show the `jetPackConfiguration` signature it should have a comment like "this will be invoked on various threads!" and then we can explain "see the comment? that's why it's incorrect" below this snippet?